sales.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head><title>Salmon Cookie</title>
+<link rel="stylesheet" href="mainstyle.css" type="text/css" />
+</head>
+<body>
+  <div id="container">
+    <div id="header">
+      <div id="logo">
+      </div>
+    </div>
+    <div id="body">
+      <div id="salesResult">
+      <table id="tableResults">
+        <div id="nav">
+        <span><a href="#" alt="Go Home">Home</a></span>
+        <span><a href="#" alt="testpage.html">test</a></span>
+        </div>
+      </table>
+      </div>
+    </div>
+    <script src="cookiesales.js"></script>
```

Comment:
This should be the last thing in the body tag.  Move this below the closing div.